docs(frontend-python): improving the doc (#1065)
zama-ai/concrete: docs/guides/manage_keys.md

```

{% hint style="warning" %}

> Do not specify the seed manually in a production environment! It's very dangerous in terms of security, it should be done only for debugging purpose.

Reviewer: ```suggestion
Do not specify the seed manually in a production environment! This is not secure and should only be done for debugging purposes.
```
